RegressionTests/projectwithconcurrency_test_output.json

```diff
@@ -10,47 +10,47 @@
           "mutationPoint" : {
             "mutationOperatorId" : "RemoveSideEffects",
             "position" : {
-              "line" : 11,
               "utf8Offset" : 317,
+              "line" : 11,
               "column" : 12
             },
-            "filePath" : "\/var\/folders\/h0\/j5sld7ss2kl3cycjxfh6nx6c0000gn\/T\/TemporaryItems\/(A Document Being Saved By muter 59)\/ProjectWithConcurrency\/ProjectWithConcurrency\/main.swift"
+            "filePath" : "\/var\/folders\/xy\/2gwd4xw17_x3r_x80n1djh340000gp\/T\/TemporaryItems\/(A Document Being Saved By muter 15)\/ProjectWithConcurrency\/ProjectWithConcurrency\/main.swift"
```

Comment:
What happened here? What does explain this change?

Reply:
The regression tests got updated, nothing important